ui/app/components/ui/boxed-message/boxed-message.component.js

```diff
@@ -0,0 +1,48 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+
+export default class BoxedMessage extends PureComponent {
+  static propTypes = {
+    boxClass: PropTypes.string,
+    iconSource: PropTypes.string,
+    iconSize: PropTypes.string,
+    style: PropTypes.object,
+    children: PropTypes.node,
+  }
+
+  static defaultProps = {
+    iconSize: '14px',
+    iconSource: null,
+    boxClass: '',
+    style: {},
+  }
+
+  render () {
+    const {
+      iconSize,
+      iconSource,
+      style,
+      boxClass,
+      children,
+    } = this.props
+
+    return (
+      <div
+        className={`boxed-message__container ${boxClass}`}
```

Comment:
Nit: Maybe it would be better to use `classNames` here? It's not a huge deal but I can't help but notice that trailing space :eyes: